Testing works well with nose too.
We do not have nose2 packaged yet.

--- semantic_version-2.10.0/setup.cfg.orig
+++ semantic_version-2.10.0/setup.cfg
@@ -36,7 +36,7 @@
 
 [options.extras_require]
 dev = 
-	nose2
+	nose
 	tox
 	check_manifest
 	coverage
--- semantic_version-2.10.0/Makefile.orig
+++ semantic_version-2.10.0/Makefile
@@ -47,7 +47,7 @@
 
 # DOC: Run tests for the currently installed version
 test:
-	python -Wdefault -m nose2
+	python -Wdefault -m nose
 
 # DOC: Perform code quality tasks
 lint: check-manifest flake8
